conf/php.ini

```diff
@@ -1,2 +1,7 @@
-memory_limit = PHP_MEMORY_LIMIT
+memory_limit = 1G
 date.timezone = "UTC"
+
+;; Zend Loader Configuration
+zend_extension = /usr/local/etc/php/zend/opcache.so
+zend_extension = /usr/local/etc/php/zend/ZendGuardLoader.so
+always_populate_raw_post_data = -1
```

Comment:
Thank you, I misread the documentation. Apparently this line is still valid for PHP 5.6. I'll make sure it's only added to the 5.6 images then.